src/Liip/RMT/Command/BaseCommand.php

```diff
@@ -34,7 +35,11 @@ public function run(InputInterface $input, OutputInterface $output)
         // Store the input and output for easier usage
         $this->input = $input;
         $this->output = $output;
-        $this->output->setDialogHelper($this->getHelperSet()->get('dialog'));
+        $dialogHelper = class_exists('Symfony\Component\Console\Helper\QuestionHelper')
```

Comment:
@jeanmonod how much do you trust the test suite? I basically just did this change to fix things without really testing it manually as well